workers/unity/Packages/io.improbable.gdk.deploymentlauncher/DeploymentLauncherWindow.cs

```diff
@@ -246,7 +246,7 @@ private void OnGUI()
                     {
                         if (GUILayout.Button(style.EditRuntimeVersionButtonContents, EditorStyles.miniButton, GUILayout.ExpandWidth(false)))
                         {
-                            GdkToolsConfigurationWindow.ShowWindow();
+                            SettingsService.OpenProjectSettings("Project/GDK Tools Configuration");
```

Comment:
This string is now hardcoded in 2 locations, should be moved to a constant variable on the provider to avoid issues